packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/UnstakeModal/UnstakeEthForm/UnstakeEthForm.tsx

```diff
@@ -106,19 +97,53 @@ export const UnstakeEthForm = () => {
                 <Divider />
             </DividerWrapper>
 
-            <UnstakeFees />
-
-            <UpToDaysWrapper>
-                <GreyP>
-                    <Translation id="TR_STAKE_UNSTAKING_PERIOD" />
-                </GreyP>
-                <Translation
-                    id="TR_UP_TO_DAYS"
-                    values={{
-                        count: unstakingPeriod,
-                    }}
-                />
-            </UpToDaysWrapper>
+            <Column gap={spacings.lg} alignItems="normal" hasDivider>
+                <UnstakeFees />
+
+                <Column gap={spacings.md} alignItems="normal" margin={{ bottom: spacings.lg }}>
+                    <Row justifyContent="space-between">
+                        <Paragraph typographyStyle="body" variant="tertiary">
+                            <Translation id="TR_STAKE_UNSTAKING_PERIOD" />
+                        </Paragraph>
+                        <Translation
+                            id="TR_UP_TO_DAYS"
+                            values={{
+                                count: unstakingPeriod,
+                            }}
+                        />
+                    </Row>
+
+                    {shouldShowInstantUnstakeEthAmount && (
+                        <Row justifyContent="space-between">
+                            <Row gap={spacings.xxs}>
+                                <Paragraph typographyStyle="body" variant="tertiary">
+                                    <Translation
+                                        id="TR_STAKE_UNSTAKING_APPROXIMATE"
+                                        values={{
+                                            symbol: symbol.toUpperCase(),
+                                        }}
+                                    />
+                                </Paragraph>
+
+                                <Tooltip
+                                    maxWidth={328}
+                                    content={
+                                        <Translation id="TR_STAKE_UNSTAKING_APPROXIMATE_DESCRIPTION" />
+                                    }
+                                >
+                                    <Icon name="info" size={14} />
+                                </Tooltip>
+                            </Row>
+
+                            <ApproximateEthAmount
```

Comment:
better name also for component

Reply:
renamed, https://github.com/trezor/trezor-suite/pull/12122/commits/9e1a100bab253b89e9c8af0fbe2e3fb7e20f24aa